shared/js/background/redirect.es6.js

```diff
@@ -97,25 +97,27 @@ function handleRequest (requestData) {
          * Click to Load Blocking
          * If it isn't in the tracker list, check the clickToLoad block list
          */
-        const socialTracker = trackerutils.getSocialTracker(requestData.url)
-        if (tracker && socialTracker && trackerutils.shouldBlockSocialNetwork(socialTracker.entity, thisTab.site.url)) {
-            if (!trackerutils.isSameEntity(requestData.url, thisTab.site.url) && // first party
-                !thisTab.site.clickToLoad.includes(socialTracker.entity) && // clicked to load once
-                !trackerutils.socialTrackerIsAllowedByUser(socialTracker.entity, thisTab.site.domain)) {
-                // TDS doesn't block social sites by default, so update the action & redirect for click to load.
-                tracker.action = 'block'
-                if (socialTracker.redirectUrl) {
-                    tracker.action = 'redirect'
-                    tracker.reason = 'matched rule - surrogate'
-                    tracker.redirectUrl = socialTracker.redirectUrl
-                    if (!tracker.matchedRule) {
-                        tracker.matchedRule = {}
+        if (thisTab.site.isFeatureEnabled('clickToPlay')) {
```

Comment:
In trackerutils we were making a check to if the feature is globally enabled but not checking the allowlists.

The code in trackerutils still checks both urls against the allowlist also; I think there is no risk in doing both.